go/vt/vtctl/reparentutil/planned_reparenter.go

```diff
@@ -177,22 +177,21 @@ func (pr *PlannedReparenter) preflightChecks(
 			event.DispatchUpdate(ev, "current primary is different than tablet to avoid, nothing to do")
 			return true, nil
 		}
+	}
 
-		event.DispatchUpdate(ev, "searching for primary candidate")
-
-		opts.NewPrimaryAlias, err = ChooseNewPrimary(ctx, pr.tmc, &ev.ShardInfo, tabletMap, opts.AvoidPrimaryAlias, opts.WaitReplicasTimeout, opts.TolerableReplLag, opts.durability, pr.logger)
-		if err != nil {
-			return true, err
-		}
-
-		if opts.NewPrimaryAlias == nil {
-			return true, vterrors.Errorf(vtrpc.Code_INTERNAL, "cannot find a tablet to reparent to in the same cell as the current primary")
-		}
+	event.DispatchUpdate(ev, "searching for primary candidate")
```

Comment:
People might be confused by the word `searching` if they see this in the logs after they've supplied `--new_primary`. Suggestion:

```suggestion
  	event.DispatchUpdate(ev, "electing a primary candidate")
```

Reply:
Done